pkg/kubelet/api/v1alpha1/runtime/api.proto

```diff
@@ -228,14 +228,22 @@ message PodSandboxConfig {
     repeated PortMapping port_mappings = 5;
     // Key-value pairs that may be used to scope and select individual resources.
     map<string, string> labels = 6;
-    // Annotations is an unstructured key value map that may be set by external
-    // tools to store and retrieve arbitrary metadata. There are a few features are
-    // driven by annotations, Runtimes could support them optionally:
+    // Unstructured key value map that may be set by external tools to store
+    // and retrieve arbitrary metadata. Annotations MUST NOT be altered or in
+    // any way processed by the runtime.
+    //
+    // In order to preserve a well-defined interface between the kubelet and
+    // the container runtime, annotations SHOULD NOT influence runtime behaviour.
+    // For legacy reasons, there are two annotations which currently explicitly
+    // break this rule, listed below; in future versions of the interface these
+    // will be promoted to typed features. Runtime authors should similarly
+    // consider proposing new fields for any new features instead of using
+    // annotations.
     //
     // 1. AppArmor
     //
     //    key: container.apparmor.security.beta.kubernetes.io/<container_name>
-    //    description: apparmor profile for the container.
+    //    description: apparmor profile for every container created in this pod.
```

Comment:
This is incorrect. The container name at the end of the annotation key specifies the container it applies to. The annotations are only on the pod since we don't have annotations on containers.